### PR TITLE
Added prettier to eslint so it shows prettier errors in eslint

### LINKED ISSUE #49: Add prettier to eslint
---
PR: https://github.com/pmedianetwork/eslint-config/pull/48

---

Currently we have two sources of truth when it comes to code formatting. One is eslint. The other one is prettier.

If we integrate prettier with eslint, we get single source of truth with better IDE hinting and single command to perform all the checks.

It is a small change that will make developer life easier.

### PR DESCRIPTION
**Pull request by adamgajzlerowicz**
Closes #49

### CODE REVIEW

**Review by danielbartsch — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "eslint-plugin-import": "2.18.2",
     "eslint-plugin-jsx-a11y": "6.2.3",
     "eslint-plugin-no-only-tests": "2.3.1",
+    "eslint-plugin-prettier": "^3.1.2",
```
Comment (TkDodo) [inline package.json]: please use exact versions:

```suggestion
    "eslint-plugin-prettier": "3.1.2",
```

File: typescript.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 module.exports = {
   "extends": [
-    "prettier/@typescript-eslint"
+    "prettier/@typescript-eslint",
+    "plugin:prettier/recommended"
```
Comment (TkDodo) [inline typescript.js]: sine we want this for every project, I think this needs to go to `index.js` instead of `typescript.js`

**Review by TkDodo — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (adamgajzlerowicz) [reply package.json]: Definitely!

File: typescript.js (hunk 2 above)
Comment (adamgajzlerowicz) [reply typescript.js]: Agreed!

File: package.json (hunk 1 above)
Comment (adamgajzlerowicz) [reply package.json]: Updated

File: typescript.js (hunk 2 above)
Comment (adamgajzlerowicz) [reply typescript.js]: Updated

**Review by TkDodo — APPROVED** [review]

**Comment by TkDodo** [comment]
DO NOT MERGE because @reakaleek is going to introduce `changesets` to this repository and automatic release management. See: http://handbook.adverity.net/changesets/

This means we will need to add a `changeset` to this PR.

**Pull request merged**